components/servo/lib.rs

```diff
@@ -269,8 +269,6 @@ impl Servo {
             .store(opts.debug.disable_share_style_cache, Ordering::Relaxed);
         style::context::DEFAULT_DUMP_STYLE_STATISTICS
             .store(opts.debug.dump_style_statistics, Ordering::Relaxed);
-        style::traversal::IS_SERVO_NONINCREMENTAL_LAYOUT
-            .store(opts.nonincremental_layout, Ordering::Relaxed);
```

Comment:
Oh, whoops, I missed this somehow. This value gets used by the style crate, and that's definitely read: https://github.com/search?q=repo%3Aservo/stylo%20is_servo_nonincremental_layout&type=code . Sorry about that!